src/cm_editor.ml

```diff
@@ -0,0 +1,107 @@
+open Import
+open Interop
+
+module Cm_document : sig
+  include CustomDocument.T
+
+  val content : t -> (string, string) result Promise.t
+
+  val onDidChange : t -> Js.Unit.t Event.t
+
+  val create : uri:Uri.t -> t
+end = struct
+  include CustomDocument
+  module OnDidChange = Event.Make (Js.Unit)
+
+  include
+    [%js:
+    val onDidChange : t -> OnDidChange.t [@@js.get]
+
+    val create :
+      uri:Uri.t -> onDidChange:OnDidChange.t -> dispose:(unit -> unit) -> t
+      [@@js.builder]]
+
+  let content t =
+    let uri = uri t in
+    let file_path = Uri.fsPath uri in
+    let bin = Path.of_string "ocamlobjinfo" in
```

Comment:
Shouldn’t this binary be invoked within a sandbox environment? 

(I’ll have time to have a full review this Friday if the PR can wait)

Reply:
Ah yes, I think you're right. I'll make the changes 👍

> (I’ll have time to have a full review this Friday if the PR can wait)

No problem, it can definitely wait 😁

Reply:
I made the change in https://github.com/ocamllabs/vscode-ocaml-platform/pull/798/commits/e881317633ebabbf9638f6c19f056b078c8c1c39